### ISSUE TITLE
Simplify egghead-ui releasing

### ISSUE THREAD

**Issue by trevordmiller** [eggheadio/egghead-instructor-center#121]
While working on the Instructor Center, it would be nice to be able to quickly make changes to egghead-ui.

The related Codeship project is already wired up to deploy the style guide app to heroku (styleguide.egghead.io) when `master` is updated, so doing the same thing for the library (https://www.npmjs.com/package/egghead-ui) makes sense. Since semver isn’t a big deal for this library, how about we additionally add a “Custom Script" deploy pipeline which publishes a new package version, always with a minor tag? Something like:
```
yarn && yarn release -- minor
```
`yarn release` runs `np`, which takes care of bumping git tags, npm package versioning, and publishing to npm. _The codeship project would need to be authorized to commit the updated versioned code to the repo and publish to the npm account._

Then the entire project workflow would look like this:

- create feature branch off of master
- `yarn` to install latest packages
- `yarn dev:styleguide` to develop using the style guide
- `yarn dev:library` to develop the library
- [localhost:2000](http://localhost:2000) to view the style guide
- use [`yarn link`](https://yarnpkg.com/lang/en/docs/cli/link/) to test using library components in another project
- submit a pull request for feature branch to master
- once continuous integration passes and there is an approved review, merge the pull request
- continuous deployment runs in Codeship and deploys the latest style guide app to [styleguide.egghead.io](https://styleguide.egghead.io) and then updates the library to a new version and publishes it [on npm](https://www.npmjs.com/package/egghead-ui)
- notify consumers to run `yarn add egghead-ui@latest` in their projects to get latest, with a list of changes

## Implementation

- [x] Update the `egghead-ui` README with the new workflow.
- [x] Add .npmrc and storybook-static/ to gitignore
- [x] Move values into environment variables
- [x] Remove the release script and dependency from the project.
- [x] Get working locally
- [x] Get deploy working; ensure styleguide.egghead.io is new, github and npm both show updates for tags + version.
- [x] Re-enable protected master branch: https://github.com/eggheadio/egghead-ui/settings/branches/master; ensure everything still works with protected branches re-enabled
- [x] Go through full process with another story to make sure it is working

**Comment by trevordmiller** [comment]
PR in the works for this: https://github.com/eggheadio/egghead-ui/pull/33

**Comment by trevordmiller** [comment]
~~Waiting on Codeship reply about the `System` errors that have no debugging info: https://helpdesk.codeship.com/hc/en-us/requests/11052~~
